ui/chart/src/acpl.ts

```diff
@@ -7,24 +7,30 @@ const Colors = {
   mistake: '#cc9b00',
 };
 
-const acpl: Window['LichessChartGame']['acpl'] = async (data: any, mainline: any[], trans: Trans, el: ChartElm) => {
+const acpl: Window['LichessChartGame']['acpl'] = async (
+  data: any,
+  mainline: Tree.Node[],
+  trans: Trans,
+  el: ChartElm
+) => {
   await loadHighcharts('highchart');
-  acpl.update = (d: any, mainline: any[]) =>
+  acpl.update = (d: any, mainline: Tree.Node[]) =>
     el.highcharts && el.highcharts.series[0].setData(makeSerieData(d, mainline));
 
   const lightTheme = window.Highcharts.theme.light;
   const blurs = [toBlurArray(data.player), toBlurArray(data.opponent)];
+  const isMod = blurs[0].length != 0 || blurs[1].length == 0;
```

Comment:
ig that should be a `!=` both times. but adjusted it to work like the movetime chart now and just use the hunter param so that it works even when there are no blurs since it probably would make it annoying to see whether there are any at a glance otherwise.

Reply:
indeed, thanks